### PR TITLE
[Poro] Added linear elastic CL with initial stresses

### PR DESCRIPTION
**Pull request by ipouplana**
Summary of contents:

- Added history linear elastic 3D and 2D laws
- Added initial stress utilities
- Cleanup and update of the python scripts involved

### CODE REVIEW

File: applications/PoromechanicsApplication/custom_constitutive/history_linear_elastic_3D_law.cpp [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+namespace Kratos
+{
+
+int HistoryLinearElastic3DLaw::Check(const Properties& rMaterialProperties,const GeometryType& rElementGeometry,const ProcessInfo& rCurrentProcessInfo)
+{
+    int ierr = BaseType::Check(rMaterialProperties,rElementGeometry,rCurrentProcessInfo);
+    if(ierr != 0) return ierr;
+
+    if ( INITIAL_STRESS_TENSOR.Key() == 0 )
+        KRATOS_THROW_ERROR( std::invalid_argument,"INITIAL_STRESS_TENSOR Key is 0. Check if all applications were correctly registered.", "" )
```
Comment (GuillermoCasas) [inline applications/PoromechanicsApplication/custom_constitutive/history_linear_elastic_3D_law.cpp]: Are you sure this is the correct error message?

File: applications/PoromechanicsApplication/custom_utilities/initial_stress_2D_utilities.hpp [hunk 2]
```diff
@@ -0,0 +1,434 @@
...
+            initial_stresses_mdpa << it_node->Id() << " " <<
+                it_node->X0() << " " <<
+                it_node->Y0() << " " <<
+                it_node->Z0() << std::endl;
+        }
+        initial_stresses_mdpa << "End Nodes" << std::endl << std::endl;
+
+        int NElems = static_cast<int>(rCurrentModelPart.Elements().size());
+        ModelPart::ElementsContainerType::iterator el_begin = rCurrentModelPart.ElementsBegin();
+        // NOTE: we are assuming that only one type of element is used (not quadratic elements)
```
Comment (GuillermoCasas) [inline applications/PoromechanicsApplication/custom_utilities/initial_stress_2D_utilities.hpp]: Then please, reflect the fact than only linear elements are assumed in the name of the method

**Review by GuillermoCasas — DISMISSED** [review]

**Review by GuillermoCasas — COMMENTED** [review]
See comments

File: applications/PoromechanicsApplication/custom_constitutive/history_linear_elastic_3D_law.cpp (hunk 1 above)
Comment (ipouplana) [reply applications/PoromechanicsApplication/custom_constitutive/history_linear_elastic_3D_law.cpp]: You are right, I'll change it by KRATOS_CHECK_VARIABLE_KEY

File: applications/PoromechanicsApplication/custom_utilities/initial_stress_2D_utilities.hpp (hunk 2 above)
Comment (ipouplana) [reply applications/PoromechanicsApplication/custom_utilities/initial_stress_2D_utilities.hpp]: Ok, thanks

**Review by GuillermoCasas — APPROVED** [review]

**Pull request merged**